menu.txt

```diff
@@ -1,3 +1,3 @@
 dosa
-karam
-oil
\ No newline at end of file
+extra - karam
```

Comment:
do we really need extra spicy

Reply:
> do we really need extra spicy

yes it allows to great taste to customers